packages/components/package.json

```diff
@@ -71,6 +71,7 @@
         "@storybook/addon-links": "^8.1.5",
         "@storybook/blocks": "^8.1.5",
         "@storybook/manager-api": "^8.1.5",
+        "@storybook/preset-scss": "^1.0.3",
```

Comment:
vite support scss by default and the plugin was necessary when storybook was using webpack instead of vite

Reply:
removed, it was a chache issue